model_analyzer/result/model_config_measurement.py

```diff
@@ -291,6 +291,43 @@ def _calculate_weighted_score(self, other):
 
         return weighted_score
 
+    def calculate_weighted_percentage_gain(self, other):
+        """
+        Calculates the weighted percentage between two 
+        ModelConfig measurements based on the weighted
+        metric objectives
+        
+        Parameters
+        ----------
+        other : ModelConfigMeasurement
+            set of (non_gpu) metrics to be compared against
+            
+        Returns
+        -------
+        float
+            The weighted percentage gain. A positive value indicates 
+            this ModelConfig measurement is better than the other
+        """
+
+        weighted_pct = 0.0
+        for objective, weight in self._metric_weights.items():
+            self_metric = self.get_metric(tag=objective)
+            other_metric = other.get_metric(tag=objective)
+
+            # Handle the case where objective GPU metric is queried on CPU only
+            if self_metric and other_metric is None:
```

Comment:
The percentage gain/loss (100) is arbitrary. Not sure what I should do in this case. Should I throw an error?

Reply:
If both are none, it seems like there is no gain, so 0% would be fine?

Reply:
Yes, I'm already setting that case to 0%, just wondering what I should be doing for the other two cases (where only one is None)

Reply:
Oh, yeah I misread the code. Is that even possible? Wouldn't the case in the comment hit the None+None case?

I'm fine with 100/-100